x/stablestake/types/tvl.go

```diff
@@ -0,0 +1,13 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	ptypes "github.com/elys-network/elys/x/parameter/types"
+)
+
+func TVL(ctx sdk.Context, accountKeeper AccountKeeper, bankKeeper BankKeeper) sdk.Int {
+	stableStakePoolAddress := accountKeeper.GetModuleAddress(ModuleName)
```

Comment:
This can be updated to authtypes.NewModuleAddress(ModuleName) and won't require accountKeeper.

Reply:
Fixed.